test-d/ElementHandle.test-d.ts

```diff
@@ -8,7 +8,7 @@ import {expectNotType, expectType} from 'tsd';
 
 declare const handle: ElementHandle;
 
-{
+async function runFirst() {
```

Comment:
why was the change in tests needed?

Reply:
I am looking into it it started to complain about top-level await. not sure what the cause is - maybe something on Chromium-BiDi side we didn't caught?

Reply:
I am not sure why this PR caused it but in short `tsd` tried to parse this files from TS to JS as CommonJS using tsc wrapper - my guess is there is complains as it see a top level await and those are forbidden in CJS.
Trying to find a repo without the change

Reply:
Seem like a `tsd` quirk.